packages/sync/src/utils/helpers.ts

```diff
@@ -35,3 +36,9 @@ export const isOnlineOnly = (op: Operation) => {
 export const isNetworkError = (error: any) => {
   return !error.result;
 };
+
+export const getMutationName = (mutation: DocumentNode) => {
+  const definition = mutation.definitions.find(def => def.kind === "OperationDefinition");
+  const operationDefinition = definition && definition as OperationDefinitionNode;
+  return operationDefinition && operationDefinition.name && operationDefinition.name.value;
```

Comment:
is it possible if the operation doesn't have a name?

Reply:
Acording to spec - Not for mutation. Operation is generic and handles queries, subscriptions and mutations. Queries can exist without explicity name but this is fairly uncommon thing anyway.